File: ios/NativeLog.m
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(NativeLog, NSObject)

+ (BOOL)requiresMainQueueSetup
{
    return NO;
}

RCT_EXTERN_METHOD(log:(NSString *)message)

RCT_EXTERN_METHOD(logWithTag:(NSString *)tag message:(NSString *)message)

@end

File: ios/NativeLog.swift
@objc(NativeLog)
class NativeLog: NSObject {

    @objc(log:)
    func log(message: String) -> Void {
        NSLog("%@", message)
    }

    @objc(logWithTag:message:)
    func logWithTag(tag: String, message: String) -> Void {
        NSLog("[%@] %@", tag, message)
    }
}
